site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,466 @@
+subhead: >
+  Ready for the next generation of web content
+date: 2021-06-14
+authors:
+  - chrishtr
+hero: 'image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/Hdfm4k1cSwfJpffgx3eH.jpg'
+alt: >
+  A pile of different devices.
+---
+
+I'm Chris Harrelson,
+the engineering lead for Rendering in Blink.
+I've been deep in the trenches of rendering performance on the web for over eight years,
+with a personal goal of doing whatever I can to make delivering excellent UX on the web faster,
+more reliable, and easier.
+In this blog post and many more to come,
+I'm excited to tell you about what we've done in that time to make Chromium's rendering engine cutting-edge.
+It's been an enormous labor of love from many people—
+me and my team as well as others that you'll also hear from. I hope you enjoy it!
+
+In 2021, we will largely complete the process of designing, building and shipping RenderingNG,
+a next-generation rendering architecture for Chromium.
+This architecture has been in progress for almost 8 years,
+and represents the collective work of many dedicated developers.
+It unlocks a huge amount of potential for the next generation of fast, fluid, reliable,
+responsive and interactive web content.
+It's also a baseline that we hope defines a new minimum standard for all web rendering engines that web developers can rely on.
+
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/kK620nxnffgeKaLLV9TJ.jpg",
+alt="Sketch of the different elements of RenderingNG",
+width="800", height="789" %}
+
+This blog post is the first in a series,
+where we'll explain what we built, why we built it, and how it works.
+
+In this first post, I'll start with:
+
+- Our North Star goal.
+- The pyramid of success:
+principles that guide our work, and examples of those principles in practice.
+- The features and capabilities that are now possible.
+- A high-level overview of the major project components of RenderingNG.
+
+## North star
+
+The north star goal motivating RenderingNG is that the browser engine implementation,
+and the richness of its rendering APIs, should not be a limiting factor of the UX of your web apps.
+
+You should not need to worry about browser bugs making features unreliable,
+or breaking your site's rendering.
+
+There should be no mysterious performance cliffs,
+with the browser slowing down your web app.
+
+You should not need to work around missing built-in features that would make UX development easy.
+
+It should Just Work.
+
+I believe RenderingNG is a huge step towards this north star goal.
+Before RenderingNG, we could (and did) add rendering features and improve performance,
+but struggled to make those features reliable for developers,
+and there were many performance cliffs.
+Now we have an architecture that systematically squashes many of those problems,
+and also unblocks advanced features that were not considered feasible before.
+
+- ✔️ Core features are rock solid across different platform/device/OS combos.
```

Comment:
The current draft post has checkmarks *and* circular bullets. Remove the circular bullets in favor of a custom bullet character.

Reply:
I don't know if that's possible - I did look at the available components to see if we had a way to do that. I'll ask the team.

Reply:
So, I spoke to the team and we don't have a method of including custom bullets. I was also asked to remove the checkmarks as they are a problem for accessibility - sorry not to be able to be more helpful there!